python/kvikio/_lib/kvikio_cxx_api.pxd

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2021-2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Reply:
This change cannot be done because it will be overwritten by the pre-commit hook with the original text before change